test/data/css/customProperties.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <style>
+      .test-custom-props {
+          --prop1: val1;
+          --prop2:val2;
+          --prop3:val3 ;
+          --prop4: "val4" ;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="test-custom-props"></div>
+    <script src="../../jquery.js"></script>
+    <script src="../iframeTest.js"></script>
+    <script>
+      startIframeTest( jQuery( ".test-custom-props" ) );
```

Comment:
I guess you got me there :), i think this is the area where we could expand and improve, since now we no longer supported oldIE, we can put those styles through `style` element in `qunit-fixture` div, which should be clear up after test run?